core/src/main/resources/org/apache/struts2/interceptor/debugging/console.ftl

```diff
@@ -21,7 +21,9 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <script type="text/javascript">
+    <script type="text/javascript" <#rt/>
+    <#include "/${parameters.templateDir}/simple/nonce.ftl" />
```

Comment:
Should the changes being made in this PR be collapsed into single lines (where possible) to match the visual structure of the original tag(s) ?

For example in this case, utilize a single line such as:
`<script type="text/javascript" <#include "/${parameters.templateDir}/simple/nonce.ftl" /> >`

to replace the three lines would eliminate the need for extra `<#rt/>` tag processing in the template and produce a visual result closer to the original tag.